crates/handlers/src/rate_limit.rs

```diff
@@ -66,15 +81,25 @@ type KeyedRateLimiter<K> = RateLimiter<K, DashMapStateStore<K>, QuantaClock>;
 
 #[derive(Debug)]
 struct LimiterInner {
+    account_recovery_per_requester: KeyedRateLimiter<RequesterFingerprint>,
+    account_recovery_per_email: KeyedRateLimiter<String>, // TODO lowercased?
```

Comment:
It could indeed have some kind of normalisation going on, but fine for now

Reply:
heh actually I already did this